packages/generator-langium/templates/packages/extension/langium-quickstart.md

```diff
@@ -0,0 +1,37 @@
+# Langium VS Code Extension
+
+Welcome to your Langium VSCode extension. This folder contains all necessary files for your language extension.
+
+## What's in the folder?
+
+- [package.json](./package.json) - The manifest file in which you declare your language support.
+- [tsconfig.json](./tsconfig.json) - The packages specific TypeScript compiler configuration extending the [base config](../../tsconfig.json).
+- [esbuid.mjs](esbuild.mjs) - Configuration file for esbuild that is used to create the VSCode extension bundle.
+- [language-configuration.json](language-configuration.json) - the language configuration used in the VS Code editor, defining the tokens that are used for comments and brackets.
+- [src/language/main.ts](src/language/main.ts) - The entry point of the language server process.
+- [src/extension/main.ts](src/extension/main.ts) - The main code of the extension, which is responsible for launching a language server and client.
+
+## Get up and running straight away
+
+- Run `npm run langium:generate` to generate TypeScript code from the grammar definition.
+- Run `npm run build` to compile all TypeScript code.
+- Press `F5` to open a new window with your extension loaded.
+- Create a new file with a file name suffix matching your language.
+- Verify that syntax highlighting, validation, completion etc. are working as expected.
+- Run `node ./bin/cli` to see options for the CLI; `node ./bin/cli generate <file>` generates code for a given DSL file.
```

Comment:
Minor: This is a CLI specific option

Reply:
Thanks, I will move that to the cli README.

Reply:
Done ⬇️